doc/user/content/cloud/administer-workspace.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Administer a Workspace"
+description: "Add or remove users to a Materialize Cloud workspace and set up workspace security."
+disable_toc: true
+menu:
+  main:
+    parent: "cloud"
+    weight:
+aliases:
+  - edit-deployments
+---
+
+{{< cloud-notice >}}
+
+Administrative users can set up shared workspaces and add and remove other users.
+
+To change a deployment name or size, click on the dots menu and select **Edit**.
```

Comment:
This is just an edit button on the detail page now, not hidden behind a dots menu.